packages/ui/src/layout/StackLayout.tsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import cx from 'classnames';
+import '@ferlab/style/layout/StackLayout.scss'
+
+export enum StackOrientation {
+    Vertical = 'vertical',
+    Horizontal = 'horizontal',
+}
+
+interface IExtraProps {
+    onClick?: (e: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => null;
+    onKeyDown?: (e: React.KeyboardEvent<HTMLElement>) => null;
+}
+
+export interface IStackLayoutProps extends IExtraProps {
+    center?: boolean;
+    children: React.ReactNode;
+    className?: string;
+    fitContent?: boolean;
+    flexContent?: boolean;
+    orientation?: StackOrientation;
+    vertical?: boolean;
+    horizontal?: boolean;
+    style?: React.CSSProperties
+}
+
+const StackLayout = ({
```

Comment:
Doing this tou can remove children from interface since React.FC has already that prop optionnal.
I feel its more cleaner than having the interface in the parenthese for the props and the return type

dont mind the space in the < and > its just because github does not allow me all glued together

const StackLayout: React.FC< IStackLayoutProps > = ({
    center,
    children,
    className,
    fitContent = false, // stretch layout to fit content
    flexContent = false,
    horizontal,
    onClick,
    orientation,
    vertical,
    style
}) => {....}

Reply:
fixed in next update